pages/form/TravelDays.jsx

```diff
@@ -134,7 +148,11 @@ export default function TravelDays() {
             <AlertTitle>Incomplete selection</AlertTitle>
           </Flex>
           <AlertDescription>
-            Please select at least one work day for each travel method.
+            {!checkIfNotAllTravelMethodsSelected
+              ? `Please select a travel method for ${daysNotYetSelected().map(
+                  (day) => ` ${day}`
+                )}.`
```

Comment:
Hi @OlaShabalina ,

minor suggestion: you could simplify this code snippet by using `.join()` instead of `map()`?

```suggestion
              ? `Please select a travel method for ${daysNotYetSelected().join(", ")}.`
```